transition-scripts/values/values-golddr-c6af30-dev-active.yaml

```diff
@@ -17,3 +17,9 @@ patroni:
 
   persistentVolume:
     size: 1Gi
+
+  podDisruptionBudget:
+    enabled: true
```

Comment:
why is this one enabled?

Reply:
I don't think it is a problem having the policy enabled in the sandbox.